dirac-dataset/src/dirac_dataset/data.py

```diff
@@ -128,63 +128,36 @@ def generate_dataset(
             p.advance(pdf_task)
 
     # -------------------------------------------------------------------------
-    # load PDFs, documentation, and issues/PRs using loader.py
+    # Load PDFs, documentation, and issues/PRs using loader.py
     # -------------------------------------------------------------------------
-    console.print("\n[bold blue]Loading PDFs and documentation...[/]")
-    pdf_docs = loader.pdf_loader(pdf_tmp)
-    md_docs = []
-    issue_and_pr_docs = []
+    console.print("\n[bold blue]Loading PDFs and documentation, inserting into Milvus...[/]")
+
+    # PDFs
+    pdf_insert_count = loader.pdf_loader(pdf_tmp)
+
+    # Repos
+    doc_insert_total = 0
+    issue_insert_total = 0
+
     with _progress() as p:
         doc_task = p.add_task("[cyan]Loading docs/issues/PRs", total=len(repos))
         for repo in repos:
             try:
-                md_docs.extend(loader.doc_loader(repo.url, branch=repo.branch))
-                issue_and_pr_docs.extend(loader.git_metadata_loader(repo.url))
+                doc_insert_total += loader.doc_loader(repo.url, branch=repo.branch)
+                issue_insert_total += loader.git_metadata_loader(repo.url)
```

Comment:
I think we can remove `loader.git_metadata_loader`. 
We will use the official Github MCP server to get issues and PRs.
Have a look at: https://github.com/github/github-mcp-server

Why I think we should use an MCP server instead of populating the DBs with the issues and PRs:
- the agent stays always up to date (it can always fetch the latest issues and PRs)
- we would not need to feed the DB with the new issues and PRs over time (because we can have about 5-10 new issues/PRs per week)

Let me know if you want further details about MCP Servers.
Do you have any opinion?